<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T6_QTClass</class>
 <widget class="QMainWindow" name="T6_QTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>T6_QT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>随机生成数字</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>直接插入排序</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>希尔排序</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>冒泡排序</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>快速排序</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>选择排序</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>堆排序</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>归并排序</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>基数排序</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>开始排序</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>10个样本部分元素</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>10个样本排序后的前20个元素</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="T6_QT.qrc"/>
 </resources>
 <connections/>
</ui>
